lib/pivotal-tracker/proxy.rb

```diff
@@ -0,0 +1,29 @@
+class BasicObject #:nodoc:
```

Comment:
Do we need the `BasicObject` class?  See built in one:  https://ruby-doc.org/core-2.2.0/BasicObject.html